src/LottieRenderer.ts

```diff
@@ -0,0 +1,327 @@
+import {
+	CompLottieLayer,
+	SpriteLottieLayer,
+	Tools
+} from './core';
+import { MeshBatcher } from './MeshBatcher';
+import { Matrix, Quaternion, Texture2D, Vector3 } from "oasis-engine";
+import { Script } from "oasis-engine";
+import { LottieResource } from './LottieResource';
+
+export { LottieLoader } from './LottieLoader';
+
+export class LottieRenderer extends Script {
```

Comment:
You can use the built-in `MeshRenderer` and a custom script `LottieAnimation` which extends `Script`, a little like `MeshRender`+ `Animator`.

In fact the 'LottieRenderer' you implement is just a lottie animation controller, so LottieAnimation is better!